microsetta_interface/templates/admin_requests_account_removal_list.jinja2

```diff
@@ -0,0 +1,97 @@
+{% extends "sitebase.jinja2" %}
+{% set page_title = _("Requests for Account Removal") %}
+{% set show_breadcrumbs = False %}
+{% block content %}
+<script>
+        function verifyDeleteRequest() {
```

Comment:
This seems nearly duplicated in the prior template, is it possible to centralize the method?

Reply:
Or maybe there's something I'm misunderstanding

Reply:
Yeah they're pretty similar, for the admin one the prompt wording is slightly different as well as the `getElementById`